insalan/tournament/management/commands/populate.py

```diff
@@ -144,6 +146,15 @@ def generate_tournaments(self, events, games):
                     game=Game.objects.get(id=game_id),
                     name="tourney_" + generate_garbage(randint(18, 26)),
                     rules=generate_garbage(randint(40000, 45000)),
+                    maxTeam=randint(1, 10),
+                    logo=self.generate_logo(),
+                    is_announced=randint(0, 100) < 50,
+                    player_price_online=randint(0, 50),
```

Comment:
prices are not `int`, they are `Decimal`. Are you sure the conversion is done when this is assigned?

Reply:
It was in my testing but it could be better for testing to have Decimal, i will change it soon :+1: